### ISSUE TITLE
SiamFCppMultiTempTracker

### ISSUE THREAD

**Issue by lfxx** [MegviiDetection/video_analyst#187]
how to use this class?i don't find its usage in your sot demo

**Comment by MARMOTatZJU** [comment]
Just search `SiamFCppMultiTempTracker` in project and you will get the corresponding config file, 

for example, `experiments/siamfcpp/test/vot/multi_temp/siamfcpp_googlenet-multi_temp.yaml`

**Comment by lfxx** [comment]
> Just search `SiamFCppMultiTempTracker` in project and you will get the corresponding config file,
> 
> for example, `experiments/siamfcpp/test/vot/multi_temp/siamfcpp_googlenet-multi_temp.yaml`

Thanks for your ripadly reply.So is our sot_video_demo.py only for single target track?we need to modify it by ourself to use multi targets track?

**Comment by lzx1413** [comment]
Yes, sot_video_demo is only for SOT process now. MultiTempTracker means sot with multi-template strategy.

**Comment by oliver8459** [comment]
> > Just search `SiamFCppMultiTempTracker` in project and you will get the corresponding config file,
> > for example, `experiments/siamfcpp/test/vot/multi_temp/siamfcpp_googlenet-multi_temp.yaml`
> 
> Thanks for your ripadly reply.So is our sot_video_demo.py only for single target track?we need to modify it by ourself to use multi targets track?

have u solved this problem? i have the same issue and could u share your code with me, i'll appreciate a lot.
